cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -179,15 +180,16 @@
     {% set has_children = nav_item.nav_groups | count > 0 or nav_item.nav_items | count > 0 %}
 
     <li class="m-list_item {{ _classes( nav_depth, '-item' ) }}"
-        {{ 'data-js-hook=behavior_flyout-menu' if has_children else '' }}>
+        {{ 'data-js-hook=behavior_flyout-menu' if has_children else '' }}
+        {{ 'aria-haspopup=menu' if has_children else '' }}
```

Comment:
Any reason not to combine L:183 & L:184 since they're reliant on the same condition?

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4217/commits/1e8272a836025fd80dba8bafafd735a2b573b6e9